static/.htaccess

```diff
@@ -22,10 +22,14 @@ RewriteRule ^docs(/(8.0|next))?/components/modeler/desktop-modeler/element-templ
 # Added Self-Managed Modeler redirect
 RewriteRule ^docs/next/self-managed/web-modeler/installation/(.*)$ /docs/next/self-managed/modeler/web-modeler/installation/$1 [R=301,L]
 
-# Renamed the folder and the file name. This page didn't exist before 1.2. 
-RewriteRule ^docs(/(8.0|1.3|1.2|next))?/apis-clients/working-with-apis-clients/?$ /docs$1/apis-tools/working-with-apis-tools/ [R=301,L]
+# Added redirect rules for APIs/Clients to APIs/Tools
+RewriteRule ^docs/next/apis-clients(.*)$ /docs/next/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/next/apis-clients(.*)$ /optimize/next/apis-tools$1 [R=301,L]
+RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/next/apis-clients/working-with-apis-tools/ [R=301,L]
+RewriteRule ^docs/apis-clients/working-with-apis-clients/?$ /docs/apis-clients/working-with-apis-tools/ [R=301,L]
 # For a short time, we had a change that renamed the page, but not the folder. This next rule covers that.
 RewriteRule ^docs/next/apis-clients/working-with-apis-tools/?$ /docs/next/apis-tools/working-with-apis-tools/ [R=301,L]
+RewriteRule ^docs/apis-clients/working-with-apis-tools/?$ /docs/next/apis-tools/working-with-apis-tools/ [R=301,L]
```

Comment:
```suggestion
# For a short time, we had a change that renamed the page, but not the folder. This next rule covers that.
RewriteRule ^docs/next/apis-clients/working-with-apis-tools/?$ /docs/next/apis-tools/working-with-apis-tools/ [R=301,L]
# Added redirect rules for APIs/Clients to APIs/Tools
RewriteRule ^docs/next/apis-clients/working-with-apis-clients/?$ /docs/next/apis-tools/working-with-apis-tools/ [R=301,L]
RewriteRule ^docs/next/apis-clients(.*)$ /docs/next/apis-tools$1 [R=301,L]
RewriteRule ^optimize/next/apis-clients(.*)$ /optimize/next/apis-tools$1 [R=301,L]
```

@christinaausley I think this captures all the rules we need, in order from most-specific to least-specific, but I'm going to let you merge it to make sure I didn't miss anything you were working on!